scripts/RemoteTests.json

```diff
@@ -178,7 +178,8 @@
                     "Local": {
                         "1": "-conns:1 -requests:1 -runtime:30000",
                         "40": "-conns:40 -requests:20",
-                        "250": "-conns:250 -requests:7500"
+                        "250": "-conns:250 -requests:7500",
+                        "1000": "-conns:1 -requests:1000"
```

Comment:
"1000": "-conns:1 -requests:1000"

Is this a typo?

Reply:
No. It's meant to be a single connection with 1000 parallel streams, to stress test the stream lookup code path.